scripts/Assert-ValidStyle.ps1

```diff
@@ -0,0 +1,43 @@
+<##
```

Comment:
Running this script will throw errors if your style is off according to those rules.  Please run it before submitting a PR.  This will be one of the scripts running in CI.